<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventreminder</class>
 <widget class="QMainWindow" name="eventreminder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eventreminder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="conectButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>sqliteconnection</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>110</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Event Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="venueLabel">
       <property name="text">
        <string>Event Venue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="venueEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="periodLabel">
       <property name="text">
        <string>Event Period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="periodEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email Adress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>82</width>
      <height>212</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dayLabel">
       <property name="text">
        <string>day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>date</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>282</width>
      <height>363</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget"/>
     </item>
     <item>
      <widget class="QTableWidget" name="eventTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
